website/sidebars.js

```diff
@@ -100,7 +100,11 @@ module.exports = {
                     label: "Project Settings",
                     items: [
                         "project_settings/project_settings_core",
-                        "project_settings/project_settings_tray_publisher"
+                        "project_settings/project_settings_nuke",
```

Comment:
These sections were deleted because their content is in the respective admin addon sections. please don't add them back in the PR